### PR TITLE
Skipping dotnet publish-iis when migrating scripts as it is now part of the web.sdk

### PR DESCRIPTION
**Pull request by livarcocc**
Fixes https://github.com/dotnet/cli/issues/4612.

@piotrpMSFT @jgoshi @krwq

### CODE REVIEW

**Comment by livarcocc** [comment]
@dotnet-bot Test Fedora23 x64 Debug Build
@dotnet-bot Test RHEL7.2 x64 Release Build

File: test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigrateScripts.cs [hunk 1]
```diff
@@ -117,53 +134,86 @@ public void Migrating_scripts_with_multiple_commands_creates_Exec_task_for_each(
...
-                commandIndex.Should().Be(-1, "Expected command array elements to be replaced by appropriate msbuild properties");
+                commandIndex.Should().Be(
+                    -1,
+                    "Expected command array elements to be replaced by appropriate msbuild properties");
             }
         }
 
         [Fact]
-        public void Formatting_script_commands_replaces_unknown_variables_with_MSBuild_Property_for_environment_variable_support()
+        public void PublishIISCommandDoesNotGetMigratedBecauseItIsNowInTheWebSDK()
```
Comment (piotrpMSFT) [inline test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigrateScripts.cs]: Curious why changing naming style?

**Comment by piotrpMSFT** [comment]
@MattGertz

File: test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigrateScripts.cs (hunk 1 above)
Comment (livarcocc) [inline test/Microsoft.DotNet.ProjectJsonMigration.Tests/Rules/GivenThatIWantToMigrateScripts.cs]: That's what is being used in other places in the code base and that's what is being used in the SDK repo as well. So, I just started unifying it. Plus, it saves a bit of horizontal space.

**Pull request merged**